kernel/dev/vga/graphics.cpp

```diff
@@ -70,6 +75,35 @@ static void testPattern() {
     }
 }
 
+static void testAnimation(){
```

Comment:
`static void testAnimation() {`

Also, this should probably be in the `apps` directory instead of the framebuffer. The `testPatter` really was only added here as a test and will be removed once my font update is done. Do you agree, @micahswitzer?

Reply:
Yeah I'd like to keep all test/example code as far separated from the implementation as possible; `apps` seems like a good place for this at the moment. If there's some debug code that might be useful again for future debugging, then just put it behind a "feature" define.

Reply:
it makes sense